builder/create-dockerfile/src/GenFiles.php

```diff
@@ -23,21 +23,28 @@ class GenFiles
     const DEFAULT_BASE_IMAGE = 'gcr.io/google-appengine/php';
     const DEFAULT_TAG = 'latest';
     const DEFAULT_WORKSPACE = '/workspace';
+    const DEFAULT_YAML_PATH = 'app.yaml';
 
     /* @var string */
     private $workspace;
 
+    /* @var string */
+    private $appYamlPath;
+
     /**
      * Constructor allows injecting the workspace directory.
      */
     public function __construct($workspace = self::DEFAULT_WORKSPACE)
     {
         $this->workspace = $workspace;
+        $yamlPath = getenv('GAE_APPLICATION_YAML_PATH') !== false
+            ?: self::DEFAULT_YAML_PATH;
```

Comment:
Failed as expected
```
RuntimeException: The application yaml file does not exist: /tmp/GenFilesTesteLdIFk/1
```

BTW true becomes `1`!

This time the test should pass.